### PR TITLE
Datalake parquet files fix

### LINKED ISSUE #9855: Failure in Azure datalake ingestions (similar to Issue #9291)
Affected module
Does it impact the UI, backend or Ingestion Framework?
Ingestion Framework

Describe the bug
Unable to ingest the Azure datalake metadata through UI.
Similar to issue : https://github.com/open-metadata/OpenMetadata/issues/9291 But now on Azure.

To Reproduce

Screenshots or steps to reproduce
Followed the steps: https://docs.open-metadata.org/connectors/database/datalake

Expected behavior
Successful ingestion of the datalake to surface the metadata in OM.

Version:

OS: [e.g. iOS]:
Python version: Python 3.7.15
OpenMetadata version: [e.g. 0.8] 0.13.1
OpenMetadata Ingestion package version: openmetadata/ingestion:0.13.1
Additional context

Error in ingestion pipeline:
[2023-01-23, 00:09:39 UTC] {es_mixin.py:101} WARNING - **Elasticsearch search failed for query** [/search/query?q=fullyQualifiedName:ssceu_sdb_Consolidations.default.dft-fr-dev-file-sys."datafirst/Landing/Retail_&_Media/LA_REDOUTE/LA_REDOUTE_ENTRANT_AGADIR/Year=2022/Month=11/Day=22/fr_webhelp_full_tde_forms-Workflow-FAQ-Volumineux_day_20221122.csv.gz"&from=0&size=10&index=table_search_index]: **500 Server Error: Internal Server Error** for url: http://10.253.69.89:8585/api/v1/search/query?q=fullyQualifiedName:ssceu_sdb_Consolidations.default.dft-fr-dev-file-sys.%22datafirst/Landing/Retail_&_Media/LA_REDOUTE/LA_REDOUTE_ENTRANT_AGADIR/Year=2022/Month=11/Day=22/fr_webhelp_full_tde_forms-Workflow-FAQ-Volumineux_day_20221122.csv.gz%22&from=0&size=10&index=table_search_index

### PR DESCRIPTION
**Pull request by ulixius9**
### Describe your changes :

Datalake parquet files fix

also fix #9855

#
### Type of change :

- [x] Bug fix

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

@open-metadata/ingestion

### CODE REVIEW

File: ingestion/src/metadata/utils/s3_utils.py [hunk 1]
```diff
@@ -90,11 +90,13 @@ def read_parquet_from_s3(client: Any, key: str, bucket_name: str):
...
-        key=client.awsAccessKeyId,
-        secret=client.awsSecretAccessKey.get_secret_value(),
-        token=client.awsSessionToken,
-    )
+    s3_fs = s3fs.S3FileSystem()
+    if client.awsAccessKeyId and client.awsSecretAccessKeyx:
+        s3_fs = s3fs.S3FileSystem(
+            key=client.awsAccessKeyId,
+            secret=client.awsSecretAccessKey.get_secret_value(),
+            token=client.awsSessionToken,
```
Comment (ayush-shah) [inline ingestion/src/metadata/utils/s3_utils.py:98]: we might also need a check for awsSessionToken

File: ingestion/src/metadata/utils/s3_utils.py [hunk 2]
```diff
@@ -90,11 +90,13 @@ def read_parquet_from_s3(client: Any, key: str, bucket_name: str):
...
+        )
```
Comment (ayush-shah) [inline ingestion/src/metadata/utils/s3_utils.py:99]: we might also need a check for awsSessionToken

Comment (ulixius9) [reply ingestion/src/metadata/utils/s3_utils.py:99]: I tested out this scenario `client.awsSessionToken` is by default set to `None` and it is okay for `token` value to be `None`

File: ingestion/src/metadata/ingestion/source/database/datalake/metadata.py [hunk 3]
```diff
@@ -287,6 +287,7 @@ def get_tables_name_and_type(  # pylint: disable=too-many-branches
                         database_name=self.context.database.name.__root__,
                         schema_name=self.context.database_schema.name.__root__,
                         table_name=table_name,
+                        skip_es_search=True,
```
Comment (ulixius9) [inline ingestion/src/metadata/ingestion/source/database/datalake/metadata.py:290]: es lookup should not be required for filtering out tables

**Review by ayush-shah — APPROVED** [review]

**Pull request merged**